Jenkinsfile

```diff
@@ -9,6 +9,7 @@ def images = [
     [image: "amazonlinux:2",                  arches: ["aarch64"]],
     [image: "centos:7",                       arches: ["amd64", "aarch64", "armhf"]],
     [image: "centos:8",                       arches: ["amd64", "aarch64"]],          // Note: armhf (arm32) images are currently not available on Docker Hub
+    [image: "clefos:7",                       arches: ["s390x"]],
```

Comment:
Should these packages end up in the `centos` directories on download.docker.com, or in `clefos` ? If they need to go into `centos`, so we need some handling to rewrite the paths?

Reply:
Ha, you tricked me 😅 I tried to modify the Jenkinsfile to tag clefos:7 -> centos:7 and sorts of things, but ran into more problems now. I looked at [that specific build](https://ci-next.docker.com/teams-core/blue/organizations/jenkins/containerd-packaging/detail/PR-167/4/pipeline/78#step-470-log-975) with just use `image: "clefos:7"` and I found that the output is already in a `centos/7` sub folder, the BASE variable is automatically set to `centos`.
I'm changing back to the initial change.

But at least I now have rebased the PR :-)